Pilot/pilotTools.py

```diff
@@ -488,43 +489,44 @@ def __init__( self ):
       self.processors = 1
 
     # Pilot command options
-    self.cmdOpts = ( ( 'b', 'build', 'Force local compilation' ),
-                     ( 'd', 'debug', 'Set debug flag' ),
+    self.cmdOpts = ( ( 'a:', 'gridCEType=', 'Grid CE Type (CREAM etc)' ),
+                     ( 'b',  'build', 'Force local compilation' ),
+                     ( 'c',  'cert', 'Use server certificate instead of proxy' ),
+                     ( 'd',  'debug', 'Set debug flag' ),
                      ( 'e:', 'extraPackages=', 'Extra packages to install (comma separated)' ),
-                     ( 'E:', 'commandExtensions=', 'Python modules with extra commands' ),
-                     ( 'X:', 'commands=', 'Pilot commands to execute' ),
-                     ( 'Z:', 'commandOptions=', 'Options parsed by command modules' ),
                      ( 'g:', 'grid=', 'lcg tools package version' ),
-                     ( 'h', 'help', 'Show this help' ),
+                     ( 'h',  'help', 'Show this help' ),
                      ( 'i:', 'python=', 'Use python<26|27> interpreter' ),
-                     ( 'k', 'keepPP', 'Do not clear PYTHONPATH on start' ),
+                     ( 'k',  'keepPP', 'Do not clear PYTHONPATH on start' ),
                      ( 'l:', 'project=', 'Project to install' ),
+                     ( 'n:', 'name=', 'Set <Site> as Site Name' ),
+                     ( 'o:', 'option=', 'Option=value to add' ),
                      ( 'p:', 'platform=', 'Use <platform> instead of local one' ),
-                     ( 'u:', 'url=', 'Use <url> to download tarballs' ),
                      ( 'r:', 'release=', 'DIRAC release to install' ),
-                     ( 'n:', 'name=', 'Set <Site> as Site Name' ),
+                     ( 's:', 'section=', 'Set base section for relative parsed options' ),
+                     ( 'u:', 'url=', 'Use <url> to download tarballs' ),
+                     ( 'x:', 'execute=', 'Execute instead of JobAgent' ),
+                     ( 'y:', 'CEType=', 'CE Type (normally InProcess)' ),
+                     ( 'z',  'pilotLogging', 'Activate pilot logging system' ),
```

Comment:
do you by chance know why some of them has an `=` and some not? e.g.:
```
pilotLogging
```
and
```
CEType=
```

does it make a difference? 

Similarly, for the "one letter options: some have `:` and some not... does it make a difference?

Reply:
Of course! Thanks.